test/extended/cmd.sh

```diff
@@ -68,3 +68,14 @@ rm -rf $tmp
 [ "$(oc new-app test/scratchimage2 -o yaml |& tr '\n' ' ' 2>&1 | grep -E "partial match")" ]
 # success with exact match	
 [ "$(oc new-app test/scratchimage -o yaml)" ]
+
+echo "[INFO] Running env variable expansion tests"
+oc new-project envtest
+oc create -f test/extended/fixtures/test-env-pod.json
+tryuntil "oc get pods | grep Running"
+podname=$(oc get pods --template='{{with index .items 0}}{{.metadata.name}}{{end}}')
+oc exec test-pod env | grep containername=test-pod
```

Comment:
indeed.  turns out this script wasn't properly exiting w/ failure.  fixed the name and the exit issue.